Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/pcr/PCRCoronaTest.kt

```diff
@@ -10,91 +11,88 @@ import java.time.Instant
 
 @Suppress("ConstructorParameterNaming")
 data class PCRCoronaTest(
-    @SerializedName("identifier")
+    @JsonProperty("identifier")
     override val identifier: TestIdentifier,
 
-    @SerializedName("registeredAt")
+    @JsonProperty("registeredAt")
     override val registeredAt: Instant,
 
-    @SerializedName("registrationToken")
+    @JsonProperty("registrationToken")
     override val registrationToken: RegistrationToken,
 
-    @SerializedName("authCode")
+    @JsonProperty("authCode")
     override val authCode: String? = null,
 
-    @SerializedName("isSubmitted")
+    @JsonProperty("isSubmitted")
     override val isSubmitted: Boolean = false,
 
-    @SerializedName("isViewed")
+    @JsonProperty("isViewed")
     override val isViewed: Boolean = false,
 
-    @SerializedName("didShowBadge")
+    @JsonProperty("didShowBadge")
     override val didShowBadge: Boolean = false,
 
-    @SerializedName("hasResultChangeBadge")
+    @JsonProperty("hasResultChangeBadge")
     override val hasResultChangeBadge: Boolean = false,
 
-    @SerializedName("isAdvancedConsentGiven")
+    @JsonProperty("isAdvancedConsentGiven")
     override val isAdvancedConsentGiven: Boolean = false,
 
-    @SerializedName("isResultAvailableNotificationSent")
+    @JsonProperty("isResultAvailableNotificationSent")
     override val isResultAvailableNotificationSent: Boolean = false,
 
-    @SerializedName("testResultReceivedAt")
+    @JsonProperty("testResultReceivedAt")
     override val testResultReceivedAt: Instant? = null,
 
-    @SerializedName("testResult")
+    @JsonProperty("testResult")
     override val testResult: CoronaTestResult,
 
-    @SerializedName("lastUpdatedAt")
+    @JsonProperty("lastUpdatedAt")
     override val lastUpdatedAt: Instant,
 
-    @Transient override val isProcessing: Boolean = false,
-    @Transient override val lastError: Throwable? = null,
+    @JsonIgnore override val isProcessing: Boolean = false,
+    @JsonIgnore override val lastError: Throwable? = null,
 
-    @SerializedName("isDccSupportedByPoc")
-    private val _isDccSupportedByPoc: Boolean? = true,
-    @SerializedName("isDccConsentGiven")
+    @JsonProperty("isDccSupportedByPoc")
+    override val isDccSupportedByPoc: Boolean = true,
```

Comment:
`isDccSupportedByPoc` in json data provided by corona test qrcode  can be `nullable` , that why `_isDccSupportedByPoc` was introduced, I think 🧐

Reply:
Reinstantiated _isDccSupportedByPoc